packages/edit-site/src/components/sidebar-edit-mode/template-panel/index.js

```diff
@@ -115,6 +116,7 @@ export default function TemplatePanel() {
 			>
 				<TemplateAreas />
 			</PostCardPanel>
+			<PluginTemplateSettingPanel.Slot />
```

Comment:
Should we move the slot below the "Transform into:" panel to retain the same position as on the `trunk`?

Reply:
I think not because initially was below the 'template card' and that changed unintentionally.. If someone reports something, we could revisit. The goal though would be to use `PluginDocumentSettingPanel` eventually.

Reply:
Makes sense 👍